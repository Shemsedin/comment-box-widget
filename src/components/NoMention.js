import React from "react";

const NoMention = () => {
  return (
    <div className="no-suggestions">
      <textarea className={"comment"} />
      <p>
        <em>
          Error fetching user data. You can still comment but without
          autosuggestion functionality.
        </em>
      </p>
    </div>
  );
};

export default NoMention;
