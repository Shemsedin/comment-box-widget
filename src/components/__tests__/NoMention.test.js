import React from "react";
import renderer from "react-test-renderer";
import NoMention from "../NoMention";

describe("NoMention", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<NoMention />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
