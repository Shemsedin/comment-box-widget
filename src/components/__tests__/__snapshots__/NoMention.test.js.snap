exports[`NoMention snapshot renders 1`] = `
<div
  className="no-suggestions">
  <textarea
    className="comment" />
  <p>
    <em>
      Error fetching user data. You can still comment but without autosuggestion functionality.
    </em>
  </p>
</div>
`;
